config/test/e2e/minio/minio.yaml

```diff
@@ -70,4 +70,6 @@ spec:
         accessModes: [ "ReadWriteOnce" ]
         resources:
           requests:
-            storage: 1Mi
+            # Minio requires at least 1Gi free space.
+            # See https://github.com/minio/minio/issues/6795
+            storage: 2Gi
```

Comment:
Actually, I think minio ignores this anyway. It seems to pick the available space on the main disk of my minikube cluster, perhaps because the volumes are hostpaths, and there's no quota system for the FS.

Reply:
See https://github.com/minio/minio/issues/6795